core/app/models/spree/variant.rb

```diff
@@ -28,6 +28,8 @@ class Variant < Spree::Base
     attr_writer :rebuild_vat_prices
     include Spree::DefaultPrice
 
+    CONDITIONS = %w[New Refurbished Used Damaged].freeze
```

Comment:
Enumeration members
- DamagedCondition
- NewCondition
- RefurbishedCondition
- UsedCondition

Different platforms requiere different visualization of condition: 
For new following values are accepted: 

1. Google: `NewCondition` or `https://schema.org/NewCondition`
2. Facebook:  `https://schema.org/NewCondition`
3. Bing: `New`
4. Yandex: `https://schema.org/NewCondition`

We will later on offer this data in the various formats through feeds and / or structured data on the single product and category pages.

Reply:
Please add following comment: 
`# Consult the various references for structured data validate frontend-output where neccessary`